lib/main.js

```diff
@@ -473,7 +473,9 @@ function ConfigureMathJax() {
   }
   if (MathJaxConfig) {
     adjustArrays(MathJaxConfig);
-    Insert(window.MathJax,MathJaxConfig);
+    // merge the defaults into the user configuration (to sanitize)
+    var currentMathJax = Insert({},window.MathJax);
+    window.MathJax = Insert(Insert({},MathJaxConfig),currentMathJax);
```

Comment:
When looking into this, I noticed that we no longer need to configure `tex2jax`, `mml2jax`, or `asciimath2jax`, since these are no longer used (we insert the explicit `<script>` tags now, rather than delimited math).  So lines 148 to 150 can be removed.

Along with that, the `Typeset` on line 701 can be changed to `Process`, since we don't need to do the preprocessor calls that are done by `Typeset()`.  Finally, the comment about setting the proper delimiters at line 661 could be changed to `//  and set the content with the proper script type`.

I think these could be part of this PR, but I know you like to do those things separately, so that could be done as a new PR if you want.